src/modules/EventList/FilterBy/index.tsx

```diff
@@ -92,7 +92,7 @@ export const FilterByUI = <T extends string | number>(props: FilterByProps<T>):
 export const FilterBy = <T extends string | number>(props: FilterByProps<T>): ReactElement => {
     const { children, onChange, options = [], value = [] } = props;
     const [focusIndex, setFocus] = useState(-1);
-    if (onChange && options.length) {
+    if (onChange) {
```

Comment:
the default options starts with 2, and then goes to zero when the config option is set to [] - which means this hook below (useWindowListener) is added and then removed so quickly it creates an error.. I dont think it hurts to have the listener.. although I would prefer another solution if you have an idea @mmichael0413

Reply:
Hmm what if we move the `if (onChange)` to inside `useWindowListener`, that way it's always defined, but we only fire the `onChange` when `onChange && options.length`?